x/auth/vesting/types/vesting_account.go

```diff
@@ -162,18 +162,18 @@ func (bva BaseVestingAccount) Validate() error {
 }
 
 type vestingAccountYAML struct {
```

Comment:
IDK if this was necessary, but with the `sigs.k8s.io/yaml` added, the annotations are not needed